### ISSUE TITLE
Problem with conversion of quotes

### ISSUE THREAD

**Issue by rockindolphin** [Automattic/amp-wp#966]
For the Russian locale, wordpress uses wptexturize to convert opening curly double quote from symbol `'&#8220;'`  into `'&#171;'` 
enabling of the plugin breaks this behavior

**Comment by rockindolphin** [comment]
found possible solution - call `wptexturize('non empty text',false)` with some text and `$reset` set to `true `before plugin init

**Comment by westonruter** [comment]
@rockindolphin I'm not able to reproduce this problem. I created a new WP install with the AMP plugin v0.6.1. I installed and activated the ru_RU locale. I created a post as follows:

![image](https://user-images.githubusercontent.com/134745/36402904-61b22f7c-1596-11e8-9efb-dee3abb5c2ca.png)

The text is:

> Это тест на поведение "wptexturize".

The AMP version of the post is then:

![image](https://user-images.githubusercontent.com/134745/36402919-7e02c326-1596-11e8-8c3a-f6d0aafa61d8.png)

So this all seems to be working as expected.
